_docker/drupal/drupal-filesystem/web/modules/custom/rhd_assemblies/src/Service/WordpressApi.php

```diff
@@ -70,10 +68,14 @@ public function getAutocompleteContentOptions($search, $max_results) {
       $response = $request->getBody()->getContents();
       $api_results = json_decode($response);
       foreach ($api_results as $api_result) {
+        $label = Html::escape($api_result->post_title) . ' (' . $api_result->ID . ')';
```

Comment:
This is probably where we want to add the type of the result, correct?

Reply:
This is just the autocomplete for wordpress results, which are all of type post. I think there is (or should be) an issue on the board for appending the type to drupal node autocomplete.